apps/code-examples/src/app/shared/theme-selector/theme-selector.component.html

```diff
@@ -45,8 +45,25 @@
     }
   </select>
 </sky-input-box>
-<sky-checkbox
-  labelText="Enable modern v2"
-  [disabled]="themeName() === 'default'"
-  [(ngModel)]="modernV2Enabled"
-/>
+<sky-input-box
```

Comment:
I modified the theme selector on all of our internal apps to allow for enabling v2 modern or rainbow mode. It will clear when you switch to default theme.